phi/tools/confluence_tools.py

```diff
@@ -0,0 +1,139 @@
+from phi.tools import Toolkit
+from phi.utils.log import logger
+from os import getenv
+
+
+try:
+    from atlassian import Confluence
+except ImportError:
+    raise ImportError("`atlassian` not installed.Please install it using `pip install atlassian-python-api`")
+
+
+class ConfluenceTools(Toolkit):
+    def __init__(self, client_id: str, auth_token: str):
+        super().__init__(name="confluence_tools")
+        self.client_id = client_id or getenv("CONFLUENCE_CLIENT_ID")
+        self.auth_token = auth_token or getenv("CONFLUENCE_AUTH_TOKEN")
+        self.confluence = Confluence(
+            oauth2={'client_id': self.client_id, 'token': self.auth_token}
+        )
+        self.register(self.get_page)
+        self.register(self.create_page)
+        self.register(self.get_page_by_id)
+        self.register(self.update_page)
+        self.register(self.delete_page)
+
```

Comment:
Add if conditions to allow the user to adjust if they want a feature or not. Eg: https://github.com/phidatahq/phidata/blob/main/phi/tools/slack.py